Code/GraphMol/Wrap/EditableMol.cpp

```diff
@@ -97,17 +104,27 @@ struct EditableMol_wrapper {
              "Remove the specified bond from the molecule")
 
         .def("AddBond", &EditableMol::AddBond,
-             (python::arg("mol"), python::arg("beginAtomIdx"),
+             (python::arg("beginAtomIdx"),
```

Comment:
Agreed that it should be consistent and isn't. I see no need to change this back; I was just curious if you'd made the change for a reason